### DISCUSSION TITLE
Expected same MINIO_ environment variables and values (*fmt.wrapError)

### DISCUSSION THREAD

**Discussion by Beau-Chen** [minio/minio#13720] [Q&A] [answered]
I'm just getting in touch with k8s and MinIO, when I'm deploying Minio using .yaml files on Kubernetes in a distributed manner, I have encountered the following errors

```Time: 07:23:44 UTC 11/19/2021
Error: http://minio-1.minio.default.svc.cluster.local:9000/storage as has incorrect configuration: Expected same MINIO_ environment variables and values (*fmt.wrapError)
       1: cmd/server-main.go:533:cmd.serverMain()
```

How can I solve this problem and where can I view all environment variables？

tks :)

**Comment by harshavardhana** [comment, marked as answer]
> Error: http://minio-1.minio.default.svc.cluster.local:9000/storage as has incorrect configuration: Expected same MINIO_ environment variables and values (*fmt.wrapError)
>        1: cmd/server-main.go:533:cmd.serverMain()

This means that MinIO is not able to get correct environment variables from operator, most probably operator webhook is not running due tom some TLS issues.

**Reply by Beau-Chen** [reply]
Thank you for your reply :)

**Comment by prathapxsz** [comment]
Could you please let me know how did you fix it. I am getting same issue.
